cmd/observe/observe_filter_test.go

```diff
@@ -257,3 +257,16 @@ func TestInvalidIdentity(t *testing.T) {
 	require.Error(t, cmd.Flags().Parse([]string{"--to-identity", "bad"}))
 	require.Error(t, cmd.Flags().Parse([]string{"--identity", "bad"}))
 }
+
+func TestTcpFlags(t *testing.T) {
+	f := newObserveFilter()
+	cmd := newObserveCmd(viper.New(), f)
+
+	// valid TCP flags
+	flags := []string{"SYN", "syn", "FIN", "RST", "PSH", "ACK", "URG", "ECE", "CWR", "NS"}
```

Comment:
Can we add a flag combination test case (e.g. `SYN,ACK`) for the sake of completeness please?

Reply:
Yes, this should be added. Will do. Thanks.